satpy/etc/enhancements/generic.yaml

```diff
@@ -143,16 +143,6 @@ enhancements:
         min_stretch: [-25, -40, 243]
         max_stretch: [0, 5, 208]
 
-  airmass_abi:
-    standard_name: airmass
-    sensor: abi
-    operations:
-    - name: stretch
-      method: *stretchfun
-      kwargs:
-        stretch: crude
-        min_stretch: [-26.2, -43.2, 243.9]
-        max_stretch: [0.6, 6.7, 208.5]
```

Comment:
Isn't this PR about AHI ?

Reply:
Yes, but this chunk of YAML was in the generic.yaml and *only* applies to ABI. It makes more sense to me to move this to the ABI specific enhancement config; especially since I copied it for the AHI specific config.